pyproject.toml

```diff
@@ -32,6 +32,7 @@ databases = { version = "^0.4.0", extras = ["postgresql", "sqlite", "mysql"], op
 orm = { version = "^0.1.5", optional = true }
 tortoise-orm = { version = "^0.16.18", extras = ["asyncpg", "aiomysql", "aiosqlite"], optional = true }
 asyncpg = { version = "0.22.0", optional = true }
+ormar = "^0.9.8"
```

Comment:
Please make `ormar` optional dependency and add `extras` to install `fastapi-pagination` with `ormar` integration.

Reply:
This one I wanted to discuss as I don't use poetry :)
Ormar have it's own extras:
```
extras_require={
        "postgresql": ["asyncpg", "psycopg2"],
        "mysql": ["aiomysql", "pymysql"],
        "sqlite": ["aiosqlite"],
        "orjson": ["orjson"],
        "crypto": ["cryptography"]
    }
```
The sync versions are used in sqlalchemy to create tables etc. Which one should I include? And should it be my name (i.e. crypto) or package name? How does it work here?

Reply:
This should look like this:
```toml
[tool.poetry.dependencies]
ormar = { version = "^0.9.8", optional = true}

[tool.poetry.extras]
ormar = ["ormar"]
```

Also, don't forget to update `all` extras)

Reply:
OK but what about ormar extras?
Like tortoise has it's own?
`tortoise-orm = { version = "^0.16.18", extras = ["asyncpg", "aiomysql", "aiosqlite"]...`

Reply:
My bad, `tortoise-orm` extras should be removed